alpine/python/2.7/Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM python:2.7.13-alpine
 LABEL maintainer "Praekelt.org <sre@praekelt.org>"
 
 # Install libffi as it is small and means we get working cffi
-RUN apk add --no-cache \
-        libffi \
-        su-exec
+RUN apk add --no-cache libffi
 
-# Install dinit (dumb-init)
-ENV DINIT_VERSION="1.2.0" \
-    DINIT_SHA256="81231da1cd074fdc81af62789fead8641ef3f24b6b07366a1c34e5b059faf363"
+# Install su-exec and tini
 RUN set -xe; \
-    apk add --no-cache wget; \
-    wget -O dumb-init "https://github.com/Yelp/dumb-init/releases/download/v$DINIT_VERSION/dumb-init_${DINIT_VERSION}_amd64"; \
-    echo "$DINIT_SHA256 *dumb-init" | sha256sum -c -; \
-    chmod +x dumb-init; \
-    mv dumb-init /usr/bin/dumb-init; \
-    ln -s /usr/bin/dumb-init /usr/local/bin/dinit; \
-    apk del wget
+    apk add --no-cache su-exec tini; \
+    mkdir /usr/local/sbin; \
+# Link `su-exec` as `gosu` for compatibility with Debian
+    ln -s "$(which su-exec)" /usr/local/sbin/gosu; \
+# Remove the script at /usr/bin/tini warning that tini is really at /sbin/tini
+    rm /usr/bin/tini; \
```

Comment:
I guess they won't, because I see we already have similar comments in other places.